fv3core/testing/translate_fvdynamics.py

```diff
@@ -1,11 +1,26 @@
+from typing import Optional
+
 import pytest
 
+import fv3core._config as spec
 import fv3core.stencils.fv_dynamics as fv_dynamics
 import fv3core.utils.gt4py_utils as utils
 import fv3gfs.util as fv3util
 from fv3core.testing import ParallelTranslateBaseSlicing
 
 
+ADVECTED_TRACER_NAMES = [
+    "qsgs_tke",
+    "qgraupel",
+    "qrain",
+    "qliquid",
+    "qice",
+    "qsnow",
+    "qo3mr",
+    "qvapor",
+]
```

Comment:
Why are you introducing a new list of these rather than using the one used everywhere else from utils? These risk getting out of sync

Reply:
I'll update this to something like `= utils.tracer_names[:DynamicalCore.NQ]`. Still need a separate variable, since this only includes advected tracers.